docs/sources/setup/migrate/ssd-to-distributed/_index.md

```diff
@@ -0,0 +1,372 @@
+1. **Data:** No changes are required to your underlying data storage. Although data loss or corruption is unlikely, it is always recommended to back up your data before starting the migration process. If you are using a cloud provider you can take a snapshot/backup.
+1. **Configuration:** We do not account for all configuration parameters in this guide. We only cover the parameters that need to be changed. Other parameters can remain the same. **However:** if `pattern_ingesters=true` you will need to spin up `patternIngesters` before shutting down the SSD ingesters. This is primarily needed for the Grafana drilldown logs feature.
+2. **Zone Aware Ingesters:** This guide does not currently account for Zone Aware Ingesters. Our current recommendation is to either disable Zone Aware Ingesters or to consult the [Mimir migration guide](https://grafana.com/docs/helm-charts/mimir-distributed/latest/migration-guides/migrate-from-single-zone-with-helm/). Take note not all paramters are equivalent between Mimir and Loki. 
+
+## Prerequisites
+
+Before starting the migration process, make sure you have the following prerequisites:
+
+1. Access to your Kubernetes cluster via `kubectl`.
+1. Helm installed.
+
+## SSD example
+
+This example will use the following SSD deployment as a reference:
+
+```yaml
+---
+loki:
+   schemaConfig:
+     configs:
+       - from: "2024-04-01"
+         store: tsdb
+         object_store: s3
+         schema: v13
+         index:
+           prefix: loki_index_
+           period: 24h
+   storage_config:
+     aws:
+       region: eu-central-1
+       bucketnames: aws-chunks-bucket
+       s3forcepathstyle: false
+   ingester:
+       chunk_encoding: snappy
+   ruler:
+    enable_api: true
+    storage:
+      type: s3
+      s3:
+        region: eu-central-1
+        bucketnames: aws-ruler-bucket
+        s3forcepathstyle: false
+      alertmanager_url: http://prom:9093
+   querier:
+      max_concurrent: 4
+
+   storage:
+      type: s3
+      bucketNames:
+        chunks: "aws-chunks-bucket"
+        ruler: "aws-ruler-bucket"
+      s3:
+        region: eu-central-1
+
+deploymentMode: SimpleScalable
+
+# SSD
+backend:
+ replicas: 2
+read:
+ replicas: 3
+write:
+ replicas: 3
+
+# Distributed Loki
+ingester:
+ replicas: 0
+ zoneAwareReplication:
+  enabled: false
+
+querier:
+ replicas: 0
+ maxUnavailable: 0
+queryFrontend:
+ replicas: 0
+ maxUnavailable: 0
+queryScheduler:
+ replicas: 0
+distributor:
+ replicas: 0
+ maxUnavailable: 0
+compactor:
+ replicas: 0
+indexGateway:
+ replicas: 0
+ maxUnavailable: 0
+ruler:
+ replicas: 0
+ maxUnavailable: 0
+
+# Single binary Loki
+singleBinary:
+ replicas: 0
+
+minio:
+ enabled: false
+```
+
+{{< admonition type="note" >}}
```

Comment:
I would move the note above the Yaml content, because some people will jump ahead and miss it.